### ISSUE TITLE
What should the default .mat data be?

### ISSUE THREAD

**Issue by hinerm** [uw-loci/knime-curvelets#3]
I had to add a default ctfDEF.mat file to the repo to run ct-fire.

I thought this was optional, but couldn't get ct-fire to run without providing a file. You can still override this file via the KNIME node, but it's not clear how to invoke the app WITHOUT a mat file, or - if a file is required - if the file I chose is the correct default.

**Comment by yuminguw** [comment]
as we discussed, this should  be fixed by  adding the 'ctfDEF.mat' into the app package while compiling the m file.

**Comment by hinerm** [comment]
It looks like the source code we had checked out from the shared drive always tries to load the param file. Looking at the latest https://github.com/uw-loci/curvelets it seems this was fixed. So in particular we need to compile from the latest source, as soon as it's ready (when released this month?)

**Comment by yuminguw** [comment]
Yes. We can sort this out  Monday when we meet. Yes, we are planning to release the new version of ctfire and curvealign by the end of this month. We can discuss later what new features in the new release can be easily adapted to the knime nodes.
